i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md

```diff
@@ -1,143 +1,140 @@
-# 部署方式
+# Deployment method
 
-<!-- ## 集群搭建 -->
-
-## 单Master模式
+## Single Master mode
 
 :::caution
-这种方式风险较大，一旦Broker重启或者宕机时，会导致整个服务不可用。不建议线上环境使用, 可以用于本地测试。
+This mode carries a higher risk, as a restart or failure of the Broker will result in the entire service being unavailable. It is not recommended for use in online environments, but can be used for local testing.
```

Comment:
It is not recommended for use in online environments, but can be used for local testing.

---

It is not recommended in online environments, but can be used for local testing.

Reply:
@tsunghanjacktsai  I will re-submit pr and fix it